runner/src/server/views/layout.html

```diff
@@ -131,7 +131,7 @@
     {% if backLink %}
         {{ govukBackLink({
             href: backLink,
-            text: "Back"
+            text: "Go back to application overview"
```

Comment:
Hmmmm I wonder if this should be configurable rather than hardcoded?

Reply:
added a config option for this to avoid having to update every from with a back link